R/github.R

```diff
@@ -23,16 +23,51 @@ github_response <- function(req) {
   parsed
 }
 
-github_GET <- function(path, ..., pat = github_pat()) {
+github_GET <- function(path, ..., pat = github_pat(),
+                       host = "api.github.com") {
+
+  url <- paste0("https://", host, "/")
+
+  # if the host contains a path, i.e. "github.hostname.com/api/v3",
```

Comment:
Why not just always include the path in the host name? Nothing would change for public github, and you'd just need to document that internal github needs the path added.